src/app/main-api-handler.ts

```diff
@@ -53,6 +54,12 @@ ipcMain.on(apiName.symphonyApi, (event: Electron.IpcMainEvent, arg: IApiArgs) =>
             // we make use of it and update the pod version info on SDA
             windowHandler.updateVersionInfo();
             break;
+        case apiCmds.registerLogRetriever:
+            registerLogRetriever( event.sender, arg.logName );
+            break;
+        case apiCmds.logReceiver:
+            finalizeLogExports( arg.logs );
```

Comment:
Please remove extra spaces for arguments

Reply:
+1